ardupilot_gz_bringup/config/iris_bridge.yaml

```diff
@@ -14,12 +14,12 @@
   ros_type_name: "nav_msgs/msg/Odometry"
   gz_type_name: "gz.msgs.Odometry"
   direction: GZ_TO_ROS
-- ros_topic_name: "tf"
+- ros_topic_name: "gazebo/tf"
```

Comment:
```suggestion
- ros_topic_name: "gz/tf"
```

Reply:
Suggest using `gz` rather than `gazebo`?

Reply:
I agree with the `gz` change.

Reply:
I agree as well, will submit the new changes